go/mysql/utils.go

```diff
@@ -83,12 +83,27 @@ func GetMasterKeyFromSlaveStatus(connectionConfig *ConnectionConfig) (masterKey
 		return nil, err
 	}
 	err = sqlutils.QueryRowsMap(db, `show slave status`, func(rowMap sqlutils.RowMap) error {
+		// Using reset slave not reset slave all, the Master_Log_File is empty.
+		if rowMap.GetString("Master_Log_File") == "" {
+			return nil
+		}
+
+		// If not using reset slave or reset slave all and slave is break, something is wrong, so we stop.
+		if rowMap.GetString("Slave_IO_Running") != "Yes" || rowMap.GetString("Slave_SQL_Running") != "Yes" {
+			log.Fatalf("The slave %s is break: Slave_IO_Running:%s Slave_SQL_Running:%s please make sure the replication is correct.",
+				connectionConfig.Key.Hostname,
+				rowMap.GetString("Slave_IO_Running"),
```

Comment:
only `GetString()` once for each column, into a variable, then use that variable.